src/crimson/os/seastore/onode.h

```diff
@@ -26,7 +26,7 @@ struct onode_layout_t {
   // snapshots.
   // TODO: implement flexible-sized onode value to store inline ss_attr
   // effectively.
-  static constexpr int MAX_SS_LENGTH = 1;
+  static constexpr int MAX_SS_LENGTH = 35;
```

Comment:
Let's add a code comment about what 35 means, point to the specfic struct related.

Like the above `MAX_OI_LENGTH` pointing to `object_info_t`.